light_curves/multiband_lc.ipynb

```diff
@@ -676,6 +676,14 @@
     "            has_icecube = True\n",
     "            # We deal with this later. Need to wait for all the things to plot\n",
     "            # so we know the y limits.\n",
+    "            # However, we have to get a dummy legend (\"lh\"), just in case this\n",
+    "            # has not been defined before... in the rare case that an object\n",
+    "            # has no data except from IceCube. There is a trick, to plot it\n",
+    "            # but set \"visible\" to False. That way it's invisible but the \n",
+    "            # legend is created. Make sure to set label=\"_nolegend_\" so that\n",
+    "            # the legend is skipped!\n",
+    "            lh = axes[\"A\"].errorbar(band_lc_clean.time , np.repeat(y , len(band_lc_clean.time)) , yerr=dy, uplims=True ,\n",
```

Comment:
This line is giving me:
```
NameError: name 'y' is not defined
```

I think `y` isn't defined until later in this cell. So maybe this should be moved down?

Reply:
Don't see this in current code. Probably already fixed in another push?

Reply:
no, found it. Was looking at the wrong version.

Reply:
I did a `git pull`, but I still see it in line 94 of this cell?

Reply:
Oh, just saw your above comment

Reply:
fixed!